api/python/notebooks/requirements.txt

```diff
@@ -0,0 +1,3 @@
+cell-census
+scikit-misc 
+scvi-tools
```

Comment:
I also install `jupyter`. While we might expect that jupyter is already installed for those commonly using notebooks, a pre-existing Jupyter installation outside of this venv prevent cell-census from being found in the python path.

Reply:
I was thinking about that. I don't think Jupyter is mandatory for running the notebooks though (for instance, I think you can also run them with vscode).